app/views/bulk_update/trainees/details/show.html.erb

```diff
@@ -0,0 +1,55 @@
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(
+    text: t(:back),
+    href: request.referrer.include?(
+      bulk_update_trainees_details_path(@bulk_update_trainee_upload)
+    ) ? request.referrer : bulk_update_trainees_uploads_path
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop" >
+    <h1 class="govuk-heading-l">
+      Your new trainees have been registered
+    </h1>
+
+    <dl class="govuk-summary-list govuk-summary-list--no-border">
```

Comment:
Can we use the component instead :pray: 

https://govuk-components.netlify.app/components/summary-list/